pkg/modules/libreoffice/routes.go

```diff
@@ -47,6 +48,7 @@ func convertRoute(libreOffice libreofficeapi.Uno, engine gotenberg.PdfEngine) ap
 				Bool("singlePageSheets", &singlePageSheets, false).
 				Bool("losslessImageCompression", &losslessImageCompression, false).
 				Bool("reduceImageResolution", &reduceImageResolution, true).
+				Bool("exportNotesInMargin", &exportNotesInMargin, false).
```

Comment:
Could you also move this before `LosslessImageCompression`?

Reply:
Done